docs/getting-started/install.md

```diff
@@ -26,16 +26,116 @@ Athena's Javascript should be added before the closing <code>&lt;/body&gt;</code
 &lt;script type="text/javascript" src="/js/framework.min.js"&gt;&lt;/script&gt;
 </code></pre>
 
-<hr>
-## CDN Coming Soon
-**Looking for a CDN link?** The Athena Framework will be available through a CDN soon.
-<hr>
+___
 
 ## Source Files
+
+Athena's source files are available in the `/src/` folder
+
 ### NPM
-If you are using NPM with your project, you can add Athena as a dependency by running <code>npm install athena-framework --save</code> in the root of your project, or by adding Athena it to your package.json file and running <code>npm install</code>.
+If you are using <a href="https://www.npmjs.com/">NPM</a> with your project, you can add Athena as a dependency by running `npm install athena-framework --save` in the root of your project, or by adding Athena it to your package.json file and running `npm install`. Make sure you also include jQuery and Tether.
 
 **package.json**
 <pre><code>{
-  "athena-framework": "github:UCF/Athena-Framework"
-}</code></pre>
\ No newline at end of file
+  &hellip;
+  "dependencies": {
+    "jquery": "^3.2.1",
+    "tether": "^1.4.0"
+    "athena-framework": "github:UCF/Athena-Framework"
+  }
+}</code></pre>
+
+### Bower
+If you are using the <a href="https://bower.io/">Bower</a> package manager, you can add Athena as a depency by running `bower install athena-framework --save` or by adding Athena to you `bower.json` file and running `bower install`. As with NPM, ensure you also include jQuery and Tether.
```

Comment:
Let's update the npm and bower install commands to install from the github repo instead of a registered project name (until we get the framework registered with those package managers), e.g. `npm install UCF/Athena-Framework --save`; `bower install UCF/Athena-Framework --save`